sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -117,22 +129,37 @@ $border-color-info: #6554c0;
 .formio-component-content h2 strong {
   font-style: normal;
   font-weight: $formio-header-wt;
-  font-size: $font-header-big;
-  line-height: $formio-header-lh-lg;
+  font-size: $font-header-large;
+  line-height: $line-height-header-lg;
 }
 
 .formio-component-content h3 strong {
   font-style: normal;
   font-weight: $formio-header-wt;
   font-size: $font-header-medium;
-  line-height: $formio-header-lh-md;
+  line-height: $line-height-header-md;
 }
 
 .formio-component-content h4 strong {
   font-style: normal;
   font-weight: $formio-header-wt;
   font-size: $font-header-small;
-  line-height: $formio-header-lh-sm;
+  line-height: $line-height-header-sm;
+}
+
+/** Forcing line height for category-header-large **/
+.formio-component-htmlelement.category-header-large p {
```

Comment:
When you have an opportunity, can you paste a print or elaborate on why these two new styles were added?

Reply:
Except for header typography styles, the line height is made common in our previous code. this common line height is 24px. 
Common line height is overriding the header large medium and small and making it to 24px. 

This part of code is added to styles following category-header-large medium and small to have their respective line height.